database/queries/manual_queries.sql

```diff
@@ -5,7 +5,7 @@
 -- issue - Hotfix 161.
 update websites set noc='remove' where site_url = 'alanstorm.com';
 
--- Based on other permissions (approcal.txt) as of 10th November 2016.
+-- Based on other permissions (approval.txt) as of 10th November 2016.
```

Comment:
Good catch @sahilsharma011 👍 & my bad 👎 . Generally I also put my code for review for at least few hours but this was hot-fix so did it without code review. I will merge it.

However **do not shy to create issues**. It is a bug. If there is a bug of any kind (including typo/grammer), we can create issues. Create issue like `Fix typo on ... page/file.` and send PR to `development` branch from `issue-xyz` branch.

**We have policy to reject any PR coming to master branch except hot-fix to be done urgently**

However no need to rework for now, I'm merging it as an exception.